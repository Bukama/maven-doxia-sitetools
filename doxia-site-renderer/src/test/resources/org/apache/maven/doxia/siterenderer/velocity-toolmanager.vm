$esc.html('Maven 2 & 3')
$display.truncate("This is a long string.", 10)
