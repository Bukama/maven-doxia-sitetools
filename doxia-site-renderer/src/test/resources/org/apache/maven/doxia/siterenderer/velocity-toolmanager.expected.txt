Maven 2 &amp; 3
This is...
